webpack.config.js

```diff
@@ -39,13 +36,16 @@ function createConfig(isDebug) {
     config.debug = true;
   } else {
     config.output.filename = 'enigma.min.js';
-    config.plugins.push(new Webpack.optimize.UglifyJsPlugin());
+    config.plugins.push(new Webpack.optimize.UglifyJsPlugin({
```

Comment:
This change seems to have caused that we no longer produce an `dist/enigma.min.js` file.